### DISCUSSION TITLE
docker run error!!!

### DISCUSSION THREAD

**Discussion by c2006878** [nocodb/nocodb#7182] [Q&A]
I am now starting the container by running a command (docker run -d --name nocodb-mysql-v2 -v "$(pwd)"/nocodb:/usr/app/data/ -p 8080:8080 -e NC_DB="mysql2://host.docker.internal:3306?u=root&p=123456&d=ceshi_noco" -e NC_AUTH_JWT_SECRET="569a1821-0a93-45e8-87ab-eb857f20a010" nocodb-local) on the Ubuntu virtual machine and reporting the following error. How should I solve it?

![1701933875187](https://github.com/nocodb/nocodb/assets/58558960/ef5039d8-b37f-4df8-a9c5-21b27c730727)

**Comment by mertmit** [comment]
Hi @c2006878,
On linux hosts you need to pass `--add-host=host.docker.internal:host-gateway` to your command in order to bind host ip to host.docker.internal
It is not automatically binded as in other operating systems.
